src/index.d.ts

```diff
@@ -95,11 +95,21 @@ export class World {
      * Gets the target of a relationship. For example, when a user calls
      * `world.target(entity, ChildOf(parent))`, you will obtain the parent entity.
      * @param entity Entity
+     * @param index Target index
      * @param relation The Relationship
      * @returns The Parent Entity if it exists
      */
     target(entity: Entity, relation: Entity, index: number): Entity | undefined;
 
+    /**
+     * Gets the target of a relationship. For example, when a user calls
+     * `world.target(entity, ChildOf(parent))`, you will obtain the parent entity.
+     * @param entity Entity
+     * @param relation The Relationship
+     * @returns The Parent Entity if it exists
+     */
+    target(entity: Entity, relation: Entity): Entity | undefined;

```

Comment:
Making an overload seems redundant, why not just do index: number | undefined

Reply:
If I did that the following behavior would be allowed, which doesn't make much sense:

```ts
world.target(entity, relation, undefined)
```